src/raw/mod.rs

```diff
@@ -329,7 +332,7 @@ impl<T> Bucket<T> {
 }
 
 /// A raw hash table with an unsafe API.
-pub struct RawTable<T> {
+pub struct RawTable<A: Alloc + Copy, T> {
```

Comment:
Can you swap the order of the parameters here?

Reply:
Also, could you explain why the allocator is required to be `Copy`?

Reply:
Will swap the order.

I added copy because of cases like [this](https://github.com/hansihe/hashbrown/blob/master/src/raw/mod.rs#L960-L972), where two maps are created from a single one. Now that you mention it it probably makes more sense to move those bounds onto an `impl RawTable` for only what needs them. 

Would it make more sense to have the bound be `Clone`?

Reply:
I think for now the best thing would be to change it to `Clone`, so that reference counted allocators work properly. In the future we may want to look into removing the need for `Clone`.